src/map/atcommand.cpp

```diff
@@ -3829,6 +3829,9 @@ ACMD_FUNC(reload) {
 	if (strstr(command, "itemdb") || strncmp(message, "itemdb", 4) == 0) {
 		itemdb_reload();
 		clif_displaymessage(fd, msg_txt(sd,97)); // Item database has been reloaded.
+	} else if (strstr(command, "cashdb") || strncmp(message, "cashdb", 4) == 0) {
+		cashdb_reload();
+		clif_displaymessage(fd, "Cash shop database has been reloaded.");
```

Comment:
Why not use msg_txt instead of hard-coded string?

Reply:
oh, yeah. because i wrote the modification for my server and just put it into the pull request, and forgot to make additional changes. in my server bcoz im too lazy to change if rathena update msg_txt and conflicting with the changes i made.